### DISCUSSION TITLE
What about distinctive name for the app?

### DISCUSSION THREAD

**Discussion by inson1** [pear-devs/pear-desktop#1731] [General]
The app has the same name as the original app, which isnt perfect. So what about rebranding to some distinctive name?

It doesnt have be something completely different, just little bit different.
https://alternativeto.net/software/youtube-music-th-ch-/

**Comment by ArjixWasTaken** [comment]
there is no original app (on desktop), unless you mean the original service/website

**Reply by inson1** [reply]
@ArjixWasTaken yes I meant original pwa, which can also run as desktop app
